03.api.md

```diff
@@ -248,48 +437,127 @@ jerry_api_eval (const char * source_p,
                                                    false, false,
                                                    &ret_val);
 }
-{% endhighlight %}
+```
+
+**See also**
+
+- [jerry_api_value_t](#jerry_api_value_t)
+- [jerry_api_create_external_function](#jerry_api_create_external_function)
+- [jerry_external_handler_t](#jerry_external_handler_t)
+
+
+# jerry_api_acquire_value
+
+**Summary**
+
+Acquires the specified Jerry API value.
+
+For values of string and object types this acquires the underlying data, for all other types it is a no-op. The acquired pointer have to be released by [jerry_api_release_value](#jerry_api_release_value).
+
+**Prototype**
+
+```c
+jerry_api_value_t *
+jerry_api_acquire_value (jerry_api_value_t *value_p);
+```
+- `value_p` - pointer to an API value
+- return value - pointer that may be used outside of the engine
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+
+  jerry_api_value_t number_value = jerry_api_create_number_value(3.14);
```

Comment:
Is this a no-op here? The doc above says it. Then what is the purpose of this call?